uiowa/uiowa
# UIPress People View

Closes #4658 

# How to test

blt ds --site=uipress.uiowa.edu
drush @uipress.local uli
Add person with person types author, illustrator, translator, editor and photographer roles selected
Add a book node and reference said person you checked to be an author/illustrator/translator/editor/photographer in the corresponding field.
Also remove a ‘check’ by a role and make sure you can’t see book name on person node (can still reference person and link to person node)
